aws/resource_aws_db_instance.go

```diff
@@ -275,6 +275,12 @@ func resourceAwsDbInstance() *schema.Resource {
 				Default:  false,
 			},
 
+			"delete_automated_backups": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  false,
```

Comment:
This is a change from the current behaviour (though it's one I would agree with for safety purposes) - not sure if the general policy would be to keep behaviour the same unless the setting is changed explicitly or not. With this and deletion protection supported RDS management feels a lot safer.

@bflad ?